src/app/containers/Headings/index.jsx

```diff
@@ -52,7 +52,7 @@ const HeadingsContainer = ({ blocks, type }) => {
     <Blocks blocks={arrayOfFragments} componentsToRender={componentsToRender} />
   );
 
-  const headingId = 'content'; // Used for the skiplink
+  const headingId = 'headline'; // Used for the skiplink
```

Comment:
do we need an id for this now? also that comment is now outdated and needs removed to avoid confusion

Reply:
We don't need an id everywhere for sure. In some areas we might need something similar for unit tests and integration tests. I'll make some changes to address that

Reply:
I think we should just remove it